Ginger/GingerCoreNET/DataSource/LiteDB.cs

```diff
@@ -708,6 +708,12 @@ public override bool RunQuery(string query)
         public void RunQuery(string query, int LocateRowValue, string DSTableName, bool MarkUpdate = false, bool NextAvai = false)
         {
             DataTable dt = GetQueryOutput("db." + DSTableName + ".find");
+            //Add a row if the table is empty to update the value.
+            if (dt.Rows.Count == 0 && NextAvai == true)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4997035)